_posts/2024-01-31-good-bad-web-components.md

```diff
@@ -97,6 +97,12 @@ Aside from Vercel.js, there are a variety of other popular component libraries t
       <td class="no">No</td>
       <td class="no">67%</td>
     </tr>
+    <tr>
+      <td>{% indieAvatar "https://19.react.dev/" %}React 19 RC</td>
+      <td class="no">No</td>
+      <td class="no">No</td>
+      <td class="no">100%</td>
+    </tr>
```

Comment:
if [this PR will wait on React 19 release](https://github.com/zachleat/zachleat.com/pull/53#issuecomment-2339251438), then this would be the more minimal diff

```suggestion
      <td class="no">No</td>
      <td class="no">100%</td>
    </tr>
```